gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -525,6 +552,35 @@ def _wait_for_network_function_create_completion(self, context,
                        'status': network_function['status']})
             raise NodeInstanceCreateFailed()
 
+    def _wait_for_network_function_update_completion(self, context,
```

Comment:
This method is similar to the method _wait_for_network_function_create_completion.
Only logs, and raised exception class on failure are different.
The timeout for wait cfg.CONF.nfp_node_driver.service_create_timeout is same for both the methods.

Why is the timeout same ? Don't we need different timeout for update node operation, as this is just config update ?

What is the state transition(s) in update operation ?